design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+# Creative Automation: Case Studies
+
+## Emergence
+
+**Conclusion:** Creativity will be low until more systems are in place, but potential is excellent. Clear and satisfying aesthetics will be key to creating a compelling contraption. Disruptions should be tunable to ensure every player can reach an optional pressure level.  
+
+### Interesting Goals
+
+Standard factory building goals are present,
+but an emphasis on sustainability and resilience add even more goals to consider.
+
+Stronger spatial and temporal variability increase the importance of considering tradeoffs and tailoring solutions to conditions.
+
+### Distinct Paths
+
+Initial prototypes will be quite weak, simply due to scale.
+
+Adding more systems and organisms will enhance this substantially.
+
+### Low Pressure
+
+Aesthetics are well-suited to chill alien cottage-core vibes.
+Disruptions are planned as an important gameplay mechanic: need excellent tuning levers to ensure that space for creativity isn't removed.
+Cascading failures shouldn't be too severe by default.
```

Comment:
We could also reward players for having hives that survived failures (possibly cascading failures)? For example, make them events that generate more "mutation points" (or whatever)? 

See also the point you raise in comments to Tim: in Factorio, you aren't rewarded for failing, so save scumming is necessary to "feel good"

Reply:
I think it works pretty well to provide some positive effects alongside the negative ones.

For example, in CIV 6 a flood can destroy your stuff, but afterwards the land provides more food or something like that.